pkgs/by-name/zo/zotify/package.nix

```
@@ -0,0 +1,46 @@
+{
+  fetchFromGitHub,
+  python3Packages,
+  lib,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "zotify";
+  version = "0.16.13";
```

```suggestion
  version = "0.6.13";
```
https://github.com/zotify-dev/zotify/blob/5da27d32a1f522e80a3129c61f939b1934a0824a/setup.cfg#L3

---

Sorry for the typo. Fixed it.